### PR TITLE
Create custom feed to allow for breaking feeds at the more tag

### PR DESCRIPTION
**Pull request by sarahmonster**
I'm not sure if this works yet—it needs testing before it's ready to deploy to the live site and testing feeds is a little delicate. It may be sufficient to solve #104.

Functionality-wise, here's what we want: feeds should ideally show the post content up to the  tag, then a link to the full post. If an excerpt is present for the post, that should *not* be used. 

I'm hoping using filters will do the trick, but we may need to build a totally custom template. Not sure about this yet; again needs more work and testing. Different feed readers also use different parts of the feed, so that's tricky as well.

### CODE REVIEW

**Comment by sarahmonster** [comment]
Other potentially-helpful links:

https://digwp.com/2012/10/customizing-wordpress-feeds/#customize-wordpress
https://gregrickaby.com/2014/03/custom-rss-template-wordpress/

https://codex.wordpress.org/Template_Tags/the_excerpt_rss
https://codex.wordpress.org/Plugin_API/Filter_Reference/the_content_feed

**Comment by michiecat** [comment]
I tried testing this a few times on my sandbox but I didn't notice any difference...I think it's because the code hasn't been deployed (since it needs to go to an outside source). How have you usually been testing the feeds? I will test it locally next. I'm wondering if I can just look at the raw XML feed to see if it formats it in the desired fashion.

**Comment by sarahmonster** [comment]
I've been testing locally, but it should work on a sandbox as well--the altered code only redirects to feedburner if the site URL is smittenkitchen.com. Is it still redirecting to feedburner on your sandbox?

File: inc/vip-functions.php [hunk 1]
```diff
@@ -5,8 +5,10 @@
  * @package smittenkitchen
  */
 
-// Redirect the feed to Feedburner.
-vip_main_feed_redirect( 'http://feeds.feedburner.com/smittenkitchen' );
+// Redirect the feed to Feedburner, only if we're on the main site!
+if( 'https://smittenkitchen.com' !== site_url() ) {
```
Comment (michiecat) [inline inc/vip-functions.php]: Should the `!` be removed?

**Comment by michiecat** [comment]
Yes, when I tested it on http://team51test.wordpress.com, the altered code redirected to Feedburner. Looks like this is caused by line 9 in feeds.php:

`if( 'https://smittenkitchen.com' !== site_url() ) {`.

I removed the `!` on my Sandbox, and it no longer redirected on the test site. More links show up for http://team51test.wordpress.com/feed, but http://feeds.feedburner.com/Team51test still shows full post content (I'm wondering if the code needs to be deployed for the updates to be reflected on Feedburner, though).

Here's the content of the raw feed from http://team51test.wordpress.com: https://gist.github.com/michiecat/9b9093de294cf9b723b5853e8f8e0d2c

The more links DO show up there. When I removed your changes, the more links DON'T show up. So it looks like it is working at least in the raw feed!

**Comment by sarahmonster** [comment]
Oh, good heavens. Apparently my brain is a giant sieve. I've pushed a change that fixes this now. I'm going to test it a bit today as well, on my sandbox, and see how emails are sent out from WordPress.com, but it actually seems like this may do the trick.

**Pull request merged**